### PR TITLE
Customer app and various changes

### PR DESCRIPTION
**Pull request by leymytel**
This PR includes all the pages needed for the customer application:
- Start page with an option to login with a PDA
- Basic page with an option to claim data or see QR code.
- Details page with detail editing.

Also:
- Pagination for CRM data
- Update all the pages to look nicer

** Note that this PR will not include any tests, I'll add them in a new PR.

### CODE REVIEW

File: ui/elyria/src/services/c2dcrm.interface.ts [hunk 1]
```diff
@@ -10,6 +10,11 @@ export interface CrmListCustomersResponse {
   id: string;
   pda_url: string;
   status: 'new' | 'claimed';
```
Comment (mudspot) [inline ui/elyria/src/services/c2dcrm.interface.ts]: Add 1 more. Status is `new`, `claimed`, `unverified`

Elyria warned that they may have unverified records.

**Review by mudspot — APPROVED** [review]
Approved. But 1 change requested before project completion.

File: ui/elyria/src/components/CustomerClaimForm.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+  const handleCustomerClaim = async (customer: CrmListCustomersResponse) => {
+    try {
+      setError('');
+      const res = await claimCustomerData(token, { id: customer.id });
+
+      if (res.data) {
+        onCustomerClaim(res.data);
+      } else {
+        setCustomers([]);
```
Comment (Ddeak) [inline ui/elyria/src/components/CustomerClaimForm.tsx]: If the claim fails or doesn't get the expect response, you reset the customers - is this the expected outcome?

**Review by Ddeak — APPROVED** [review]
LGTM!

File: ui/elyria/src/components/CustomerClaimForm.tsx (hunk 2 above)
Comment (leymytel) [reply ui/elyria/src/components/CustomerClaimForm.tsx]: Hmm probably not! Good spot!

**Pull request merged**